deploy-addresses/unichain-sepolia.json

```diff
@@ -1,4 +1,4 @@
 {
-  "UniversalRouter": "0x5418a6d871d327a1cefe6deb97153f07cbf6030e",
-  "UnsupportedProtocol": "0x7B46ee9BaB49bd5b37117494689A035b0F187B59"
+  "UniversalRouterV2": "0xe7cea85aa4bf60fb54c1b70004e4f8414b3708fd",
+  "UnsupportedProtocol": "0x76870DEbef0BE25589A5CddCe9B1D99276C73B4e"
```

Comment:
why did this address change? and where did you get it from?

Reply:
hm no they both look wrong... old address is an EOA. New address definitely has different bytecode to the mainnet one 😅 
I'll dig into it a little

Reply:
summary after a conversation on slack:
- there is no UnsupportedProtocol we think, as no protocol has ever needed one. So remove that
- we are going to re-deploy UR to unichain sepolia because neither of these addresses has the best versions of all protocols